README.md

```diff
@@ -409,6 +419,22 @@ exports.handler = (event, context, callback) => {
 };
 ```
 
+## HTTP Exception handling
+If the logger did not receive a successful response from the server when tried to send a log,
+it will retain the logs in its buffer and will try to flush it with the next request.
+The flush limit of the line that comes after a failed one will be reset to the backoff period.
+The failed messages retention byte size is limited. The limit and back off
+period are configurable via options:
+
+``` javascript
+const options = {
```

Comment:
I recommend we be consistent throughout the file since it's public - since we're using `var` let's keep using that. If we want to use `const` and `let` we should convert everywhere.